test/controllers/auth.test.js

```diff
@@ -65,31 +65,32 @@ describe(path, () => {
       expect(status).to.equal(200);
     });
 
-    it('should return a token', async () => {
-      const { body } = await request(server)
+    it('should have Set-Cookie header', async () => {
+      const { headers } = await request(server)
         .post(path)
         .send({
           email: newUser.email,
           password: newUser.password,
         });
 
-      expect(body).to.have.property('jwt');
-      expect(body.jwt).to.match(/^([^.]+\.){2}[^.]+$/);
+      expect(headers).to.have.property('set-cookie');
+      expect(headers['set-cookie']).to.be.a('array');
+      expect(headers['set-cookie'][0]).to.be.a('string');
     });
 
-    it('should have Set-Cookie with SameSite=Strict; Secure; HttpOnly and token', async () => {
+    it('should Set-Cookie have jwt; SameSite=Strict; Secure; HttpOnly attribs', async () => {
```

Comment:
This test validates if Set-Cookie has jwt, if it was in the correct format and the other attributes. In this case is best to separate the tests in two: one being the first comment (Set-Cookie having jwt and it is correct) and the other if Set-Cookie has the other attributes.

Reply:
Done